src/components/LeftSidebar/LeftSidebar.spec.js

```diff
@@ -136,25 +136,17 @@ describe('LeftSidebar.vue', () => {
 			expect(conversationsListMock).toHaveBeenCalled()
 
 			const appNavEl = wrapper.findComponent({ name: 'NcAppNavigation' })
-			const listEl = appNavEl.findComponent({ name: 'ConversationsList' })
 
-			expect(listEl.exists()).toBe(true)
-			expect(listEl.props('searchText')).toBe('')
-			expect(listEl.props('initialisedConversations')).toBe(false)
+			// Check that all the conversations are rendered
+			const conversationsArray = appNavEl.findAllComponents({ name: 'Conversation' })
+			expect(conversationsArray.length).toBe(conversationsList.length)
```

Comment:
```suggestion
			expect(conversationsArray).toHaveLength(conversationsList.length)
```
Also applicable, and more readable